lms/djangoapps/bookmarks/tests/test_views.py

```diff
@@ -312,6 +353,29 @@ def test_unsupported_methods(self):
         self.assertEqual(405, self.client.put(reverse('bookmarks')).status_code)
         self.assertEqual(405, self.client.delete(reverse('bookmarks')).status_code)
 
+    @patch('lms.djangoapps.bookmarks.views.tracker.emit')
+    @ddt.unpack
+    @ddt.data(
+        {'page_size': -1, 'expected_bookmarks_count': 2, 'expected_page_size': 10, 'expected_page_number': 1},
+        {'page_size': 0, 'expected_bookmarks_count': 2, 'expected_page_size': 10, 'expected_page_number': 1},
+        {'page_size': 999, 'expected_bookmarks_count': 2, 'expected_page_size': 500, 'expected_page_number': 1}
+    )
```

Comment:
I'm unclear what should be returned, say, if there are three bookmarks, and one requests a page size of 2 and requests the second page.  I'm assuming the count would be 3, the page number would be 2, and the page size would be 2, and that the actual number of items shown on that page would have to be calculated.

Is it possible to include a test of this?  (i.e. beyond page one...)